tests/Integration/Database/EloquentPrimaryUuidTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Illuminate\Tests\Integration\Database;
+
+use Illuminate\Database\Eloquent\Concerns\HasPrimaryUuid;
+use Illuminate\Database\Eloquent\Model as Eloquent;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Str;
+
+class EloquentPrimaryUuidTest extends DatabaseTestCase
+{
+    protected function defineDatabaseMigrationsAfterDatabaseRefreshed()
+    {
+        Schema::create('users', function (Blueprint $table) {
+            $table->uuid('id')->primary();
```

Comment:
what about different name of the primary key? for example `uuid` then we should override `$primaryKey` property in the model to this name?

Reply:
This is just an example. You can still adjust the primary key name to whatever you want.

Reply:
If I use different name, then I'll add `HasPrimaryUuuid` and override `$primaryKey` property for model, yes?

Reply:
Yep.